src/docs/introduction.rst

```diff
@@ -1,80 +1,97 @@
+    - `sssd-ipa <https://www.mankier.com/5/sssd-ipa>`_
+    - `sssd-ad <https://www.mankier.com/5/sssd-ad>`_
+    - `sssd-files <https://www.mankier.com/5/sssd-files>`_
+
+    For more information about FreeIPA and other compatible directory servers,
+    please check out the following links.
+
+    - `389 Directory Server`_
+    - `FreeIPA`_
+    - `Microsoft Active Directory`_
+    - `OpenLDAP`_
+
+Merging Linux and Windows worlds
+********************************
+
+IPA and AD can have a trust created between the domains. Why? Consider if
+there was a company merger or acquisition. There won’t be a need to migrate
+users but instead, set up a trust and allow Bob from industry giant "SSSD Corp"
+to administer newly acquired small business "LDAP LLC".
 
 It’s not uncommon for companies to have two departments, one that supports
 and administers Linux and the other team manages Windows, so each will have
-control over its own users and groups. Using SSSD’s ipa_provider,  AD domain
+control over its own users and groups. Using SSSD’s IPA provider,  AD domain
 users can then log in to a host managed by IPA because of the established
 trust between domains.
 
-SSSD can authenticate directly to AD, using the ad_provider, and they’ll
-be no need for IPA servers. Users and groups will be entirely managed by
-AD. So why use indirect authentication? IPA is more flexible for managing
-Linux hosts than AD since it can manage SUDO rules, SELinux users and
-contexts, automount maps, and more. If all users are created in AD, but
-the company has two IT teams. The Linux team may have to wait for users
-to be created. Hopefully, you can start to see that each solution has its
-pros and cons and will not be discussed in this introduction.
-
-For more information about FreeIPA and other compatible directory servers,
-please check out the following links.
-
-- `FreeIPA <https://www.freeipa.org/page/Main_Page>`_
-- `389DS <https://directory.fedoraproject.org/>`_
-- `OpenLDAP <https://www.openldap.org/>`_
-- `Kerberos <https://web.mit.edu/kerberos/>`_
-- `Microsoft ADDS <https://www.microsoft.com/en-ie/windows-server>`_
-
-The ad_provider and ipa_provider are just some providers that can be used
-for authentication or authorization. Other providers is the ldap_provider,
-krb_provider, proxy_provider, and local_provider. For more information please
-check out the following MAN pages for more information.
-
-- sssd-ldap
-- sssd-krb5
-- sssd-ipa
-- sssd-ad
-- sssd-files
+SSSD can authenticate directly to AD and there will be no need for IPA servers.
+Users and groups will be entirely managed by AD. So why use indirect
+authentication? IPA is more flexible for managing Linux hosts than AD since it
+can manage SUDO rules, SELinux users and contexts, automount maps, and more. If
+all users are created in AD, but the company has two IT teams. The Linux team
+may have to wait for users to be created. Hopefully, you can start to see that
```

Comment:
Recommend fixing the wording of "If all users are created in AD, but the company has two IT teams. The Linux team may have to wait for users to be created."

Something like the following instead:

"If a company employs separate Linux and Windows teams, SSSD-AD direct integration may result in Linux teams waiting for AD users to be created."